backend/graminsta/graminsta/urls.py

```diff
@@ -14,8 +14,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from rest_framework.authtoken import views
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('core/', include('core.urls', namespace='core')),
+    path('core-token-auth/', views.obtain_auth_token, name='core-token-auth'),
```

Comment:
I think you can put all `authtoken` views under a unified path, like `authtoken`, so for each view, it's path is `authtoken/obtain-new-auth-token`, etc.